modular_zubbers/modules/ballstics/wt_autorifle.dm

```diff
@@ -0,0 +1,192 @@
+	ricochet_decay_damage = 0.8
+
+.//AMMO CASING
+/obj/item/ammo_casing/c46x30mm/flathead
+	name = "4.6x30mm flathead bullet casing"
+	desc = "A 4.6x30mm bullet casing."
+	caliber = CALIBER_46X30MM
+	projectile_type = /obj/projectile/bullet/c46x30mm/flathead
+
+/obj/item/ammo_casing/c46x30mm/rubber
+	name = "4.6x30mm rubber-tipped bullet casing"
+	desc = "A 4.6x30mm bullet casing."
+	caliber = CALIBER_46X30MM
+	projectile_type = /obj/projectile/bullet/c46x30mm/rubber
+
+
+//MAGAZINE
+/obj/item/ammo_box/magazine/wt550m9/flathead
+	name = "wt550 magazine (FlatHead 4.6x30mm)"
+	icon_state = "46x30mmt-20"
+	base_icon_state = "46x30mmt"
+	ammo_type = /obj/item/ammo_casing/c46x30mm/flathead
+	caliber = CALIBER_46X30MM
+	max_ammo = 20
+
+/obj/item/ammo_box/magazine/wt550m9/rubber
+	name = "wt550 magazine (Rubber 4.6x30mm)"
+	icon_state = "46x30mmt-20"
+	base_icon_state = "46x30mmt"
+	ammo_type = /obj/item/ammo_casing/c46x30mm/rubber
+	caliber = CALIBER_46X30MM
+	max_ammo = 20
+
+
+//ORDERING
+/datum/supply_pack/security/armory/wt551
+	name = "WT-551 Autorifle Crate"
+	desc = "Contains a pair of WT-551 Autorifles pre-loaded with less-than-lethal rubber-tipped rounds. Additional ammo sold seperately. Backwards-compatible with WT-550 magazines. NanoTrasen reminds you that the other weapon is for a friend, and not for going guns akimbo."
+	cost = CARGO_CRATE_VALUE * 8
+	contains = list(/obj/item/gun/ballistic/automatic/wt550/security/rubber = 2)
+	crate_name = "wt-550 autorifle crate"
+
+/datum/supply_pack/security/armory/wt550_ammo_rubber
+	name = "WT-550/WT-551 Autorifle Ammo Crate (Rubber-Tipped)"
+	desc = "Contains 3 magazines with less than lethal rubber-tipped rounds for the WT-551."
+	cost = CARGO_CRATE_VALUE * 4
+	contains = list(/obj/item/ammo_box/magazine/wt550m9/rubber = 3)
+	crate_name = "wt-550 magazine crate (rubber-tipped)"
+
+/datum/supply_pack/security/armory/wt550_ammo_flat
+	name = "WT-550/WT-551 Autorifle Ammo Crate (Flat-Tipped)"
+	desc = "Contains 3 magazines with lethal flat-tipped rounds for the WT-551."
+	cost = CARGO_CRATE_VALUE * 5
+	contains = list(/obj/item/ammo_box/magazine/wt550m9/flathead = 3)
+	crate_name = "wt-550 magazine crate (flat-tipped)"
+
+/datum/supply_pack/security/armory/wt550_ammo_regular
+	name = "WT-550/WT-551 Autorifle Ammo Crate (Regular)"
+	desc = "Contains 3 magazines with lethal flat-tipped rounds for the WT-551."
+	cost = CARGO_CRATE_VALUE * 7
+	contains = list(/obj/item/ammo_box/magazine/wt550m9 = 3)
+	crate_name = "wt-550 magazine crate (regular)"
+
+//RESEARCHING: TECHWEBS
+
+/datum/techweb_node/weaponry/New()
+	design_ids += "wt550_ammo_rubber"
+	design_ids += "wt550_ammo_flathead"
+	. = ..()
+
+/datum/techweb_node/adv_weaponry/New()
+	design_ids += "wt550_ammo_normal"
+	. = ..()
+
+/datum/techweb_node/exotic_ammo/New()
+	design_ids += "wt550_ammo_ap"
+	. = ..()
+
+/datum/techweb_node/syndicate_basic/New()
+	design_ids += "wt550_ammo_incendiary"
+	. = ..()
+
+//RESEARCHING: DESGINS
```

Comment:
Put this inside ``modular_zubbers\code\modules\research\techweb\all_nodes.dm`` I want all techweb New() procs located inside the same folder using ``design_ids += list()`` for consistency.

Reply:
can't you just do that yourself when the time comes so I don't have to deal with .dme conflicts when everyone uses that folder

Reply:
Sure